doc/leak_tracking/DETECT.md

```diff
@@ -87,7 +110,7 @@ Leak tracking is fully available.
 
 **Flutter profile**
 
-Leak tracking is available, but MemoryAllocations that listens to
+Leak tracking is available, but FlutterMemoryAllocations that listens to
```

Comment:
place FlutterMemoryAllocations in backticks. Better yet, maybe link to the api for this class